src/irmin/tree.ml

```diff
@@ -924,6 +924,17 @@ module Make (P : Backend.S) = struct
       | Error _ as e -> e
       | Ok m -> Ok (StepMap.bindings m)
 
+    let update_bindings updates bindings =
```

Comment:
`update_bindings` sounds like "let's update the bindings".

Something like `let seq_of_updates update_map value_bindings` instead?